pkg/config/config.go

```diff
@@ -65,6 +65,8 @@ type MicroshiftConfig struct {
 	NodeIP          string        `json:"nodeIP"`
 	BaseDomain      string        `json:"baseDomain"`
 	Cluster         ClusterConfig `json:"cluster"`
+	// Used to trigger a force rollout on network components upon MicroShift restart
+	CreationTimestamp string `json:"creationTimestamp"`
```

Comment:
Do we need to expose this field to users or can it be an internal value?

Reply:
It will be for internal use only. Captured when microshift main process starts.